front/lib/resources/vault_resource.ts

```diff
@@ -301,6 +347,12 @@ export class VaultResource extends BaseResource<VaultModel> {
         return auth.isBuilder() && auth.canWrite([this.acl()]);
 
       case "regular":
+        // TODO(SPACE_INFRA): Represent this in ACL.
```

Comment:
Implemented in https://github.com/dust-tt/dust/pull/8094 to mitigate the impact and keep permissions out of the noise.